src/dev/build/tasks/os_packages/docker_generator/templates/base/Dockerfile

```diff
@@ -134,7 +134,7 @@ RUN for iter in {1..10}; do \
     (exit $exit_code)
 {{/ubuntu}}
 {{#wolfi}}
-RUN apk --no-cache add bash curl fontconfig libstdc++ nss findutils shadow
+RUN apk --no-cache add bash curl fontconfig libstdc++ libnss findutils shadow
```

Comment:
We need libnss to start Chrome for reporting.  On other distributions we are installing nss which pulls this library in.  Not an issue currently on serverless, reporting is disabled.